plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/server/debug/OpenShiftDebugUtils.java

```diff
@@ -298,21 +298,20 @@ public void setupRemoteDebuggerLaunchConfiguration(ILaunchConfigurationWorkingCo
 		}
 	}
 	
-	public boolean isRunning(ILaunchConfiguration launchConfiguration, int localDebugPort) {
+	public boolean isRunning(ILaunchConfiguration launchConfiguration) {
 		boolean isRunning = getLaunches()
-				.filter(l -> !l.isTerminated() && launchMatches(l, launchConfiguration, localDebugPort))
+				.filter(l -> !l.isTerminated() && launchMatches(l, launchConfiguration))
```

Comment:
@bdshadow sure while I fully agree with the general rule as much as I would also subscribe to "remove dead code" I am reluctant in case of our OpenShift v2 code because "never touch a running system" and "we won't invest anything here" seems equally important to me. If you are completely sure that we are not breaking anything (we should maybe have QE confirming) then we can have a go for this change.